pages/utilites/astral_runner.md

```diff
@@ -37,4 +37,4 @@ Default `astral_runner.py` output:
   - A greedy consensus of the 100 bootstrapped replicate trees.
   - The “main” ASTRAL tree.
 
-**NOTE**: Users may choose to use the trees generated by `sgt_constructor.py` using the ortholog files generated by `matrix_constructor.py`. For example, `sgt_constructor.py –trees_only -i matrix_constructor_out_<M.D.Y>/trimal/ -o <above, directory of your choice>/`. If users construct trees via this method, example run is `rtc_binner.py -i <directory of your choice>/RAxML/ -st gt80trimal.tre -s gt80trimal`.
\ No newline at end of file
+**NOTE**: Users may choose to use the trees generated by `sgt_constructor.py` using the ortholog files generated by `matrix_constructor.py`. For example, `sgt_constructor.py –trees_only -i matrix_constructor_out_<M.D.Y>/trimal/ -o <above, directory of your choice>/`. If users construct trees via this method, example run is `rtc_binner.py -i sgt_constructor_out_<M.D.Y>/trees/ -st gt80trimal.tre -s gt80trimal`.
```

Comment:
Change "`rtc_binner.py -i sgt_constructor_out_<M.D.Y>/trees/ -st gt80trimal.tre -s gt80trimal`." to "astral_runner.py -i sgt_constructor_out_<M.D.Y>/trees/"

Reply:
Ah I see.